package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "custom-event-polyfill": "joscha/custom-event-polyfill#de24bd9",
     "debounce": "^1.0.0",
-    "weakmap": "0.0.6"
+    "webcomponents.js": "skatejs/webcomponentsjs#skate-v1"
```

Comment:
I doubt that will happen, but if they do, it's behaviour that will go in anyways since it's built to a spec.